pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -642,7 +642,22 @@ func BuildKubernetesMasterConfig(
 	return kmaster, nil
 }
 
-func DefaultOpenAPIConfig() *openapicommon.Config {
+func DefaultOpenAPIConfig(config configapi.MasterConfig) *openapicommon.Config {
+	securityDefinitions := spec.SecurityDefinitions{}
+	if len(config.ServiceAccountConfig.PublicKeyFiles) > 0 || config.OAuthConfig != nil {
+		securityDefinitions["BearerToken"] = &spec.SecurityScheme{
+			SecuritySchemeProps: spec.SecuritySchemeProps{
+				Type:        "apiKey",
+				Name:        "authorization",
+				In:          "header",
+				Description: "Bearer Token authentication",
+			},
+		}
+	}
+	if configapi.UseTLS(config.ServingInfo.ServingInfo) {
+		// TODO: How is this represented ?
```

Comment:
So based on http://swagger.io/specification/#securityDefinitionsObject I would have to say there is no way to assert our use of x509 😞